packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -274,6 +278,9 @@ class _AnsiRun {
 /// terminal window by default. If the stdout is not a terminal window, then the
 /// default will be [outputPreferences.wrapColumn].
 ///
+/// If [columnWidth] is clamped to [kMinColumnWidth] at minimum (so passing negative
```

Comment:
uber nit, confusing sentence. Should the `if` be removed since column width is always clamped to kMinColumnWidth

Reply:
ah yeah that is the result of some bad editing. Good catch. Will fix.